snyk/cli
# fix: Fix invalid JSON path in IaC test summary test

#### What does this PR do?

- Fixes the loading path for the IaC formatted issue fixtures from `test/jest/unit/iac/fixtures/formatted-results.json` to `test/jest/unit/iac/process-results/fixtures/formatted-results.json`

#### Where should the reviewer start?

- `test/jest/unit/lib/formatters/iac-output/test-summary.spec.ts`

#### How should this be manually tested?

- Execute the `test/jest/unit/lib/formatters/iac-output/test-summary.spec.ts` test and ensure all the tests pass.

#### Any background context you want to provide?

- Due to the invalid loading path, the test was consistently failing. 

#### Additional information

- [Example to a failing CircleCI pipeline](https://app.circleci.com/pipelines/github/snyk/cli/11466/workflows/a44960e9-9f5b-4b56-be49-f1ab98404184/jobs/88106)
